src/resources/source.rs

```diff
@@ -830,6 +978,15 @@ pub struct CreateSourceRedirect {
     pub return_url: String,
 }
 
+#[derive(Clone, Debug, Deserialize, Serialize)]
+pub struct CreateSourceSourceOrder {
```

Comment:
This struct appears twice (both `Create` and `Update`, if you "rename" each one to be the same (perhaps `SourceOrderParams`) then it will deduplicate and rename the child structs as well (`{Create,Update}SoruceSourceOrder{Items,Shipping}`).

Reply:
Addressed in comment.